README.md

```diff
@@ -465,3 +465,5 @@ Attach mode:
 # Contributing
 
 Bug reports and pull requests are welcome on GitHub at https://github.com/ruby/debug.
+
+Please also check the [contributing guideline](/CONTRIBUTING.md).
```

Comment:
@ko1 I plan to revisit the contributing guideline and this section after the tests PR is merged.